tests/console/zypper_docker.pm

```diff
@@ -48,11 +47,8 @@ sub run {
     assert_script_run("zypper-docker list-updates-container tmp_container", timeout => 600);
     assert_script_run("zypper-docker list-patches-container tmp_container", timeout => 600);
     # apply all the updates to a new_image
-    if (is_sle('>=15')) {
-        record_soft_failure 'bsc#1123173';
-    } else {
-        assert_script_run("zypper-docker update --auto-agree-with-licenses $testing_image new_image", timeout => 900);
-    }
+    # if (is_sle('>=15')) record_soft_failure 'bsc#1123173'
```

Comment:
@eliroca Mind removing this line?. It isn't needed anymore :)
```suggestion
```

Reply:
I could remove it, but @okurz made a good point in recommending we should keep the soft failure reference for a while.

Reply:
@okurz is the first person to complain when there's disabled code :). Leave a comment instead if you prefer.

Reply:
foursixnine is right :)

Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit. Having a comment in plain english is better.